WordPress/playground-tools
# Playground block: Update CodeMirror deps for a11y fix

## What?

This PR updates CodeMirror dependencies to include a recent accessibility-related autocomplete fix:
https://discuss.codemirror.net/t/avoiding-aria-autocomplete-list-when-no-autocompletion-providers/8313/2

## Why?

Fixes #289 - "Playground block: A11Y: Use correct ARIA specification for autocomplete"

## Implementation

This PR updates npm dependencies for CodeMirror. The Playground block appears to work well with the update.

The interactive-code-block doesn't render well in the editor either before or after these updates. (Is it still in use anywhere, and if not, can we delete it? cc @adamziel)

## Testing Instructions

- Run `npm install` update dependencies
- Run: `npx nx dev wordpress-playground-block` to build the block and start a dev server
- Open the editor and create a post containing the WordPress Playground block
- Leave "index.php" in place, and add a file type like "test.xyz" that is not a file type with autocomplete support
- Publish the post and view on the front end
- Open the browser dev tools
- Select "index.php", run `$$('[aria-autocomplete]')`, and confirm the query returns an element
- Select "test.xyz", run `$$('[aria-autocomplete]')`, and confirm the query returns an empty list
